charts/README.md

```diff
@@ -48,12 +48,15 @@ their default values.
 | `global.coordinatorNamespaceLabel`           | string         | Control plane label. Do not edit | `"edgeless.systems/control-plane-ns"` |
 | `global.podAnnotations`                      | object         | Additional annotations to add to all pods | `{}`|
 | `global.podLabels`                           | object         | Additional labels to add to all pods | `{}` |
-| `marbleInjector.CABundle`                    | string         | Set this to use a custom CABundle for the MutatingWebhook | `""` |
+| `marbleInjector.CABundle`                    | string         | MutatingWebhook CA bundle. Automatically configured by the MarbleRun CLI. Ignore when using standalone helm chart | `""` |
 | `marbleInjector.image`                       | string         | Name of the marbleInjector container image | `"coordinator"` |
 | `marbleInjector.start`                       | bool           | Start the marbleInjector webhook | `false` |
 | `marbleInjector.replicas`                    | int            | Replicas of the marbleInjector webhook | `1` |
 | `marbleInjector.repository`                  | string         | Name of the container registry to pull the marbleInjector image from | `"ghcr.io/edgelesssys/marblerun"` |
 | `marbleInjector.version`                     | string         | Version of the marbleInjector container image to pull | `"v1.0.0"` |
+| `marbleInjector.useCertManager`              | bool           | Set to use cert-manager for certificate provisioning. Required when using standalone helm chart for installation | `false` |
```

Comment:
We need to set this flag in the helm installation docs, right?: https://github.com/edgelesssys/marblerun/pull/372